### PR TITLE
 Komikindo.ID: fix HTTP error 404

### PR DESCRIPTION
**Pull request by AhmadGhoni**
closes https://github.com/tachiyomiorg/tachiyomi-extensions/issues/13687

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/id/komiku/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ ext {
     extName = 'Komiku'
     pkgNameSuffix = 'id.komiku'
     extClass = '.Komiku'
-    extVersionCode = 14
+    extVersionCode = 6
```
Comment (alessandrojean) [inline src/id/komiku/build.gradle:8]: Should be `15`.

```suggestion
    extVersionCode = 15
```

File: src/id/komikindoid/src/eu/kanade/tachiyomi/extension/id/komikindoid/KomikIndoID.kt [hunk 2]
```diff
@@ -54,10 +54,7 @@ class KomikIndoID : ParsedHttpSource() {
     }
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        val builtUrl = if (page == 1) "$baseUrl/daftar-komik/" else "$baseUrl/daftar-komik/page/$page/?order="
-        val url = builtUrl.toHttpUrlOrNull()!!.newBuilder()
-        url.addQueryParameter("title", query)
-        url.addQueryParameter("page", page.toString())
+        val url = "$baseUrl/daftar-manga/page/$page/?".toHttpUrlOrNull()?.newBuilder()!!.addQueryParameter("title", query)
```
Comment (alessandrojean) [inline src/id/komikindoid/src/eu/kanade/tachiyomi/extension/id/komikindoid/KomikIndoID.kt:57]:
```suggestion
        val url = "$baseUrl/daftar-manga/page/$page/".toHttpUrlOrNull()!!.newBuilder()
            .addQueryParameter("title", query)
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by AhmadGhoni** [comment]
Sorry it looks like I got the build.gradle extension wrong, it should be KomikindoID

**Comment by TheKingTermux** [comment]
Why did you change the ext VersionCode version in Komiku?
while in the KomikIndoID not?
Looks like you got it wrong in build.gradle

**Comment by AhmadGhoni** [comment]
I'm looking at the wrong folder

> > build.gradle extension wrong
> 
> Why did you change the extVersionCode version in Komiku? [5284d94](https://github.com/tachiyomiorg/tachiyomi-extensions/commit/5284d946c7466fd91534167da6db7ae8c6d59b52) while in the KomikIndoID not? https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/src/id/komikindoid/build.gradle Looks like you got it wrong in build.gradle

**Pull request merged**